nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -582,6 +582,36 @@ The following settings can be configured in _nifi.properties_ to control JSON We
 |`nifi.security.user.jws.key.rotation.period` | JSON Web Signature Key Rotation Period defines how often the system generates a new RSA Key Pair, expressed as an ISO 8601 duration. The default is one hour: `PT1H`
 |==================================================================================================================================================
 
+[[csrf-protection]]
+=== Cross-Site Request Forgery Protection
+
+NiFi 1.15.0 introduced Cross-Site Request Forgery protection as part of user interface access based on session cookies.
+CSRF protection builds on a standard Spring Security Filter and implements the double submit cookie strategy. The
+implementation strategy relies on sending a random request token cookie at the beginning of the session, and requiring
+the JavaScript user interface to send the value in a corresponding request header. NiFi applies the `SameSite`
```

Comment:
Do we need "JavaScript" at "user interface".

Reply:
The word `JavaScript` could be omitted, but I included it as a pointer to the implementation. Other implementation strategies embed CSRF request tokens in hidden HTML form parameters, but the NiFi implementation uses a custom JavaScript handler for HTTP requests.